libs/blocks/src/lib/features/card-content/index.tsx

```diff
@@ -1,39 +1,30 @@
 import clsx from 'clsx';
 import { FluidContainer, Heading, Section, Text } from '@deriv/quill-design';
 import { ReactNode } from 'react';
-import { CardContent } from '@deriv-com/components';
+import {
+  CardContent,
+  CardVariantType,
+  CardsContainer,
+} from '@deriv-com/components';
 
 export interface CardFeatureProps {
   title?: string;
   description?: string;
   cta?: ReactNode;
   className?: string;
   cards?: CardContent[];
-  renderCard?: React.FC<CardContent>;
-  rows?: 'one' | 'two';
   cols?: 'two' | 'three' | 'four';
+  cardsVariant?: CardVariantType;
 }
 
-const cardColsVariant = {
-  two: 'grid-cols-1 sm:grid-cols-2',
-  four: 'grid-cols-1 sm:grid-cols-2 lg:grid-cols-4',
-  three: 'grid-cols-1 sm:grid-cols-2 lg:grid-cols-3',
-};
-
-const cardRowsVariant = {
-  one: 'grid-rows-1',
-  two: 'grid-rows-2',
-};
-
 const CardFeature = ({
   title,
   description,
   cta,
   className,
   cards = [],
-  renderCard,
   cols = 'two',
-  rows = 'one',
+  cardsVariant = 'ContentBottom',
```

Comment:
can we keep it `variant`  I think it's already clear that it's a card.

Reply:
sure will do, thanks :pray:

Reply:
done